fpga_interchange/compare.py

```diff
@@ -0,0 +1,163 @@
+#/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+""" Implements a way to compare two FPGA interchange capnp messages. """
```

Comment:
Since this comparison utility requires unittest and presumably is used in the tests only wouldn't it be better to move it to the `tests` directory ?

Reply:
I'm going to leave it in the library for now, and we can refactor this to be useful in a command line utility in the future.